src/pmie/rc_pmie

```diff
@@ -75,19 +51,6 @@ else
     VERBOSE_CTL=off
 fi
 
-id=`id | sed -e "s/(.*//" -e "s/.*=//"`
-if [ "$id" != 0 -a "$1" != "status" ]
-then
-    if [ -n "$PCP_DIR" ]
-    then
-	: running in a non-default installation, do not need to be root
-    else
-	echo "$prog:"'
-Error: You must be root (uid 0) to start or stop pmie.'
-	exit
-    fi
-fi
-
```

Comment:
As discussed on slack, the restorecon invocation later in this file can go.

Reply:
_reboot_setup() all removed in commit 283b560b1